addons/mail/static/src/models/follower/follower.js

```diff
@@ -78,16 +66,11 @@ function factory(dependencies) {
          */
         async remove() {
             const partner_ids = [];
-            const channel_ids = [];
-            if (this.partner) {
-                partner_ids.push(this.partner.id);
-            } else {
-                channel_ids.push(this.channel.id);
-            }
```

Comment:
In this file you can also remove all the "channel*" fields at the bottom.
And also make partner field required, and change name compute to be a related to partner name instead. (_computeName can be deleted too then)

Reply:
Also _computeResId and _computeResModel can be adapted